src/swarmus_example_pkg/test/DummyUtilsTests.cpp

```diff
@@ -0,0 +1,15 @@
+#include "swarmus_example_pkg/DummyUtils.h"
+#include <gtest/gtest.h>
+
+TEST(TestSuite, dummyTestCase) { ASSERT_TRUE(true); }
+
+TEST(TestSuite, testAddTwoNumbers) {
+    int sum = DummyUtils::addTwoNumbers(2, 2);
+
+    ASSERT_EQ(sum, 4);
+}
+
+int main(int argc, char** argv) {
```

Comment:
Sometimes when running tests in ROS we need to initialise some ROS stuff (`ros::init() and such`). In the examples I followed, this was always done in the main. I should keep it there since it can prove useful.

Reply:
You could use a fixture and put it in the Setup function, that's it use. You would need to use TEST_F instead of TEST macro. [Check this out](https://github.com/google/googletest/blob/master/googletest/docs/primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests-same-data-multiple-tests) and check the hivemind if you want a concrete example.